cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -102,6 +107,29 @@ impl snowbridge_pallet_inbound_queue::Config for Runtime {
 	type AssetTransactor = <xcm_config::XcmConfig as xcm_executor::Config>::AssetTransactor;
 }
 
+impl snowbridge_pallet_inbound_queue_v2::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Verifier = snowbridge_pallet_ethereum_client::Pallet<Runtime>;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type XcmSender = PolkadotXcm;
+	#[cfg(feature = "runtime-benchmarks")]
+	type XcmSender = DoNothingRouter;
+	type GatewayAddress = EthereumGatewayAddress;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Helper = Runtime;
+	type WeightInfo = crate::weights::snowbridge_pallet_inbound_queue_v2::WeightInfo<Runtime>;
+	type AssetHubParaId = ConstU32<1000>;
+	type Token = Balances;
+	type XcmPrologueFee = ConstU128<XCM_PROLOGUE_FEE>;
+	type AssetTransactor = <xcm_config::XcmConfig as xcm_executor::Config>::AssetTransactor;
+	type MessageConverter = snowbridge_router_primitives::inbound::v2::MessageToXcm<
+		EthereumNetwork,
+		ConstU8<INBOUND_QUEUE_PALLET_INDEX>,
+		EthereumSystem,
+		ConstU128<XCM_PROLOGUE_FEE>,
```

Comment:
Passing in `XCM_PROLOGUE_FEE` into the pallet (for burning fees) and into the message converter (to specify the PayFees amount). Not sure if this is a smell and I should rather pass the fee value from the pallet to the message converter.